ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/timeline/TimelineComponentView.kt

```diff
@@ -82,39 +92,47 @@ internal fun TimelineComponentView(
                 modifier = Modifier.height(timelineState.itemSpacing.dp)
                     .constrainAs(itemSpacingRef) {
                         top.linkTo(bottomContentBarrier)
-                        if (isLastItem) {
-                            bottom.linkTo(parent.bottom)
-                        }
                     },
             )
 
-            IconComponentView(
-                style = item.icon,
-                state = state,
+            Box(
```

Comment:
Had to wrap the icon into a box so we can size the box with the size of the biggest icon, then the icon can just be center aligned.